source/arriving/remote_work.rst

```diff
@@ -0,0 +1,31 @@
+.. _ref_remote_work:
+
+Working from home
```

Comment:
In my opinion, "working from home" should be a section of "Setting up your computer", possibly named "SSH" or something.